includes/ApplHandlingByStr.php

```diff
@@ -41,6 +82,44 @@ function showStatus( $stat_str )
 }
 
 
+function needAcceptor($status)
+{
+	$str = explode ( ';', $status );
+	for ( $i =  1;  $i < count ( $str )-1 ; $i++ )
+	{
+		$st = explode ( ',', $str[$i] )[1];
+		
+		if( $st == 'P' )
+		{
+			return false;
+		}
+	}
+	
+	if( explode(',', $str[ count($str)-1 ] )[1] == 'P' )
+	{
+		return true;
+	}
+	
+	return false;
+}
+
+function needApprover($status)
+{
+	$str = explode ( ';', $status );
+	for ( $i =  1;  $i < count ( $str )-1 ; $i++ )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space found after opening bracket of FOR loop
* Space found before closing bracket of FOR loop
* Expected 1 space after first semicolon of FOR loop; 2 found
* Space found before second semicolon of FOR loop
* Expected 0 spaces after opening bracket; 1 found
* Space before opening parenthesis of function call prohibited
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 5

<!-- Comment by [SideCI](https://sideci.com) -->